src/app/pages/api-keys/components/api-key-form-dialog/api-key-form-dialog.component.ts

```diff
@@ -33,10 +27,7 @@ export class ApiKeyFormDialogComponent implements OnInit {
   constructor(
     private fb: FormBuilder,
     private dialogRef: MatDialogRef<ApiKeyFormDialogComponent>,
-    private matDialog: MatDialog,
-    private ws: WebSocketService,
-    private loader: AppLoaderService,
-    private errorHandler: FormErrorHandlerService,
+    private store$: ApiKeyComponentStore,
```

Comment:
Original `store$` had dollar sign because it was an observable. It's not needed here.

Reply:
Fixed